RdlMigration/ElementNameConstants.cs

```diff
@@ -85,10 +85,10 @@ public class DataSetConstants
         public class PowerBIWrapperConstants
         {
             public const string MyWorkspace = "My Workspace";
-            public const string ResourceUrl = "https://analysis.windows.net/powerbi/api";
```

Comment:
Revert this file. You can keep the change to edog locally just don't commit.

Reply:
reverted